src/main/scala/org/spot/SpotLDACWrapper.scala

```diff
@@ -1,4 +1,4 @@
-package org.opennetworkinsight
+package org.spot
```

Comment:
Is this the case where we need to use apache in the package name? Or is that happening after incubating process?
org.apache.spot?